File: String/1108. Defanging an IP Address.cpp
class Solution {
public:
    string defangIPaddr(string address) {
        string new_address;
        for(int i=0; i<address.size(); i++) {
			if(address[i] == '.')
            {
                new_address.push_back('[');
                new_address.push_back('.');
                new_address.push_back(']');                
            }
            else
                new_address.push_back(address[i]);
                
		}
		return new_address;  
        
        
    }
};

File: String/1108. Defanging an IP Address.py
class Solution(object):
    def defangIPaddr(self, address):
        """
        :type address: str
        :rtype: str
        """
        new_address = ""
        for data in address:
            if data == ".":
                new_address += "[.]"
            else:
                new_address += data
        return new_address
    
class Solution(object):
    def defangIPaddr(self, address):
        """
        :type address: str
        :rtype: str
        """
        return address.replace(".","[.]")